scripts/align-version.sh

```diff
@@ -13,8 +13,8 @@ files="$(find . -name package.json | grep -v node_modules | xargs)"
 ${scriptdir}/align-version.js ${files}
 
 # validation
-marker=$(node -p "require('./scripts/get-version-marker')")
-if find . -name package.json | grep -v node_modules | xargs grep "[^0-9]${marker}"; then
+marker=$(node -p "require('./scripts/get-version-marker').replace(/\./g, '\.')")
+if find . -name package.json | grep -v node_modules | xargs grep -E "\"[^0-9]?${marker}\""; then
```

Comment:
Do we really need the `^[0-9]` now that you've added quotes?

Reply:
Maybe replace with `[\^~]?` if you want at some point to match caret or tilde versions?

Reply:
Yes, I think we still need this